Highlight React import

dspaccapeli opened the pull request:
# Why

Right now the "React" import is present in the first snippet of the page where it's not used (yet) or highlighted.
A reader could easily miss the import. I would propose to move it to the second snippet when it's used and also to add an highlight to show it as "new" code.

# How

I updated the text on Github text editor.

# Test Plan

Rebuilding the app would show the new updated text.

dspaccapeli committed:
Highlight React import

Right now the "React" import is present in the first snippet of the page where it's not used (yet) or highlighted.
A reader could easily miss the import. I would propose to move it to the second snippet when it's used and also to add an highlight to show it as "new" code.

Simek commented on docs/pages/tutorial/image-picker.md:
````diff
@@ -34,7 +34,6 @@ With the library installed in our project, we can now actually use it.
 
 <!-- prettier-ignore -->
 ```js
-import React from 'react';
````
Please revert this line.

dspaccapeli replied:
Reverted in the new commit. For consistency with the previous examples I agree it is good to keep it, but it seems that it is not used or included in the create-expo-app before that.

Simek commented on docs/pages/tutorial/image-picker.md:
````diff
@@ -87,6 +86,7 @@ Now we will take the data that we get from the image picker and use it to show t
 
 <!-- prettier-ignore -->
 ```js
+/* @info Import React to use useState */import React from 'react';/* @end */
````
If you want to add imports to the second example, please list all of them.

dspaccapeli replied:
Done. I thought to remove it because in [a previous page](https://docs.expo.dev/tutorial/button) it is mentioned in the end to include only relevant code in the snippets.

Simek reviewed (changes requested):
Hello @dspaccapeli, thank you for the contribution! 👍 

I have left two suggestions, please address those and we can ship the change.

dspaccapeli committed:
Update image-picker.md

Addressed comments from review, added full imports in the second snippet and reverted the deletion in the first one

Simek commented on docs/pages/tutorial/image-picker.md:
```diff
@@ -87,6 +87,11 @@ Now we will take the data that we get from the image picker and use it to show t
+/* @info Import React to use useState */import React from 'react';/* @end */
+import { Image, StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+import * as ImagePicker from 'expo-image-picker';
+
+  
```
```suggestion

```

Simek committed:
whitespace cleanup

Simek commented on docs/pages/tutorial/image-picker.md:
````diff
@@ -87,6 +87,9 @@ Now we will take the data that we get from the image picker and use it to show t
 ```js
+/* @info Import React to use useState */import React from 'react';/* @end */
+import { Image, StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+import * as ImagePicker from 'expo-image-picker';
````
```suggestion
import * as ImagePicker from 'expo-image-picker';

```

Simek committed:
whitespace change again

Simek reviewed (approved):
Thank you for the updates! 🙏
